Mathlib/LinearAlgebra/Dimension/Finrank.lean

```diff
@@ -48,6 +48,9 @@ Defined by convention to be `0` if the space has infinite rank.
 
 For a vector space `M` over a field `R`, this is the same as the finite dimension
 of `M` over `R`.
+
+Note that it is possible to have `M` with `¬(Module.Finite R M)` but `finrank R M ≠ 0`, for example
+$$\mathbb{Z} \oplus (\mathbb{Z}/2\mathbb{Z})^{\mathbb{N}}$$ has `finrank` equal to `1`.
```

Comment:
I believe that `$$` will make it a display math, which is unnecessary I think. Also, maybe `Z \oplus Q/Z` is a simpler example?

Reply:
It's also fine to use unicode in the math for `\mathbb{Z}` and `\oplus` I think; it still renders reasonably well, and is much easier to read in the editor.

Reply:
> It's also fine to use unicode in the math

Then I think we don't need to use `$` anymore, since there are no subscripts and superscripts:

```suggestion
`ℤ ⊕ ℚ/ℤ` has `finrank` equal to `1`.
```